contracts/interchain-token/ERC20Permit.sol

```diff
@@ -73,8 +72,6 @@ abstract contract ERC20Permit is IERC20, IERC20Permit, ERC20 {
 
         if (uint256(s) > 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF5D576E7357A4501DDFE92F46681B20A0) revert InvalidS();
 
-        if (v != 27 && v != 28) revert InvalidV();
```

Comment:
i prefer to be more explicit here. also doesn't seem a big enough reason to change the InterchainToken bytecode (existing ITS tokens already have this check, and new ones won't causing an inconsistency that's not a must have)

Reply:
Had discussion, and agreed that we don't want to change the `InterchainToken bytecode`, hence closing this PR.